mclagsyncd/mclaglink.cpp

```diff
@@ -197,14 +197,14 @@ void MclagLink::setPortIsolate(char *msg)
     cur = msg;
 
     /*get isolate src port infor*/
-    op_hdr = (mclag_sub_option_hdr_t *)cur;
+    op_hdr = reinterpret_cast<mclag_sub_option_hdr_t *>(static_cast<void *>(cur));
```

Comment:
>static_cast<void *>( [](start = 56, length = 20)

Remove `reinterpret_cast<mclag_sub_option_hdr_t *>()` and only use `reinterpret_cast` ?

Reply:
Would you please, add the exact  the way you want to try? Im bit confused with the comment

Reply:
Sorry for misleading. Re-commented and added suggestion.

---
In reply to: [443787216](https://github.com/Azure/sonic-swss/pull/1325#discussion_r443787216) [](ancestors = 443787216)

Reply:
Please find error for removing "static cast"
`mclaglink.cpp: In member function 'void swss::MclagLink::setPortIsolate(char*)':
mclaglink.cpp:201:60: error: cast from 'char*' to 'mclag_sub_option_hdr_t* {aka mclag_sub_option_hdr_t_*}' increases required alignment of target type [-Werror=cast-align]
     op_hdr = reinterpret_cast<mclag_sub_option_hdr_t *>(cur);
                                                            ^
cc1plus: all warnings being treated as errors
Makefile:439: recipe for target 'mclagsyncd-mclaglink.o' failed
make[4]: *** [mclagsyncd-mclaglink.o] Error 1
`